### DISCUSSION TITLE
SASS Breaking change

### DISCUSSION THREAD

**Discussion by johnrmore** [vuetifyjs/vuetify#13880] [Releases]
Is there a plan to release an update to fix https://sass-lang.com/documentation/breaking-changes/slash-div
I am using vue 2.6.14 and vuetify 2.56 found many similar to
   ╷
62 │     'sm': $grid-gutter / 6,
   │           ^^^^^^^^^^^^^^^^
   ╵
    node_modules/vuetify/src/styles/settings/_variables.scss 62:11   @import
    node_modules/vuetify/src/styles/settings/_index.sass 1:9         @import
    node_modules/vuetify/src/styles/styles.sass 2:9                  @import
    node_modules/vuetify/src/components/VBanner/_variables.scss 1:9  @import
    stdin 2:9                                                        root stylesheet

**Comment by Aymkdn** [comment]
Answer is there: https://github.com/vuetifyjs/vuetify/issues/13694

> This is fixed in Vuetify 3, you now have to use vuetify-loader or @vuetify/vite-plugin to change variables.
> Vuetify 2 is still limited to sass@~1.32
